Sources/GRPCHTTP2Core/Internal/ContentType.swift

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2024, gRPC Authors All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// See:
+// - https://github.com/grpc/grpc/blob/master/doc/PROTOCOL-HTTP2.md
+enum ContentType {
+  case protobuf
```

Comment:
Is this naming from their docs because I find it confusing that it is `protobuf` but we send `"application/grpc"`

Reply:
I kept the same naming from v1. `application/grpc` and `application/grpc+proto` are interchangeable, so I think that's where the name comes from.